analyzer/src/funTest/assets/projects/external/spdx-tools-python-expected-output.yml

```diff
@@ -27,8 +27,47 @@ project:
     dependencies:
     - id: "PyPI::ply:3.11"
     - id: "PyPI::rdflib:5.0.0"
-    - id: "PyPI::six:1.16.0"
+      dependencies:
```

Comment:
Does the upgrade without adding the new option also affect the expected output? / would it make sense to have separate commits for (1) upgrade and (2) adding the new option?

Reply:
The option is required for some test projects, I have added this to the commit message, see: https://github.com/nexB/python-inspector/issues/72